frontend/src/components/RFI/RFILanding.vue

```diff
@@ -1426,6 +1420,7 @@ import { useApplicationStore } from '../../store/application.js';
 import { useNavBarStore } from '../../store/navBar.js';
 import { useReportChangesStore } from '../../store/reportChanges.js';
 import { useSupportingDocumentUploadStore } from '../../store/supportingDocumentUpload.js';
+import { VTimePicker } from 'vuetify/labs/VTimePicker';
```

Comment:
I've replaced the time picker on other pages with the custom component AppTimeInput, similar to what we've done in OFM. Should we use AppTimeInput instead of VTimePicker?

Reply:
on it :)